src/webview/html/messageAsHtml.js

```diff
@@ -80,6 +80,14 @@ export default (backgroundData: BackgroundData, message: Message | Outbox, isBri
      class="message ${isBrief ? 'message-brief' : 'message-full'}"
      id="msg-${id}"
      data-msg-id="${id}"
+     $!${
```

Comment:
Extract this and the similar code in `message-full` to a common function.

Reply:
Sure! Thanks.